app/views/content/what-pension-does-a-teacher-get.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
Is this where we want the pensions page to sit in the IA @philspray - under the root? Would we be better having it as a child page of Salaries and benefits (would make more sense as a breadcrumb?)?

(If so, might be something that Ross needs to help with!)

Reply:
I'm presuming this URL make sense in terms of search terms?

Reply:
So when @EllieNodder and I chatted about this, we thought it would be best under here until we do the navigation work to decide where everything sits. Open to moving it though?

The data suggests this is what people are searching for.

Reply:
OK - happy to leave this for now under the root subject to being cleared up later!